crates/protogen/proto/metastore/options.proto

```diff
@@ -120,9 +119,14 @@ message StorageOptions {
   map<string, string> inner = 1;
 }
 
+message TableOptionsV1 {
+  string name = 1;
+  bytes options = 2;
+}
```

Comment:
Seems like it'd be a good idea to add a version number here so that we can iterate on versioning on a per-data source way (without needing to rename data sources.)